client/src/components/SignupForm.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
```

Comment:
create a folder for signup and put your jsx and css file to it

Reply:
_done_